packages/fxa-admin-panel/src/components/TableYHeaders/index.tsx

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ReactElement } from 'react';
+import { HIDE_ROW } from '../../../constants';
+
+interface TableYHeadersProps {
+  header?: string;
+  testId?: string;
+  borderL?: boolean;
+  children:
+    | ReactElement<TableRowYHeaderProps>
+    | ReactElement<TableRowYHeaderProps>[];
+}
+
+interface TableRowYHeaderProps {
+  header: string | ReactElement;
+  value?: null | string | ReactElement | ReactElement[];
+  testId?: string;
+  className?: string;
+}
+
+export const TableRowYHeader = ({
+  header,
+  value,
+  testId,
+  className,
+}: TableRowYHeaderProps) => {
+  if (!value || value === HIDE_ROW) {
+    return null;
+  }
+
+  return (
+    <tr {...{ className }}>
+      <th className="table-th text-left">{header}</th>
+      <td data-testid={testId} className="table-td border-b">
```

Comment:
That makes sense, I made `TableXHeaders` after I'd done `TableYHeaders` and can see now how it can probably be more general with the same interface. By "provide a set of elements," do you mean the way `td` is provided?

Reply:
I was just thinking a list (i.e. set) of TD elements for each row.